### PR TITLE
Cleanup log messages in Core.

### PR DESCRIPTION
**Pull request by praveek**
## Description

Cleanup log messages in Core. 

## Related Issue
https://github.com/adobe/aepsdk-core-android/issues/196

## Types of changes

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)

## Checklist:

- [ ] I have signed the [Adobe Open Source CLA](http://opensource.adobe.com/cla.html).
- [ ] My code follows the code style of this project.
- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.
- [ ] I have read the **CONTRIBUTING** document.
- [ ] I have added tests to cover my changes.
- [ ] All new and existing tests passed.

### CODE REVIEW

File: code/android-core-library/src/main/java/com/adobe/marketing/mobile/internal/eventhub/EventHub.kt [hunk 1]
```diff
@@ -136,14 +138,6 @@ internal class EventHub {
 
     init {
         registerExtension(EventHubPlaceholderExtension::class.java) {}
-        registerExtension(ConfigurationExtension::class.java) {
```
Comment (yangyansong-adbe) [inline code/android-core-library/src/main/java/com/adobe/marketing/mobile/internal/eventhub/EventHub.kt:139]: Do we register the configuration extension in another place?

File: code/android-core-library/src/phone/java/com/adobe/marketing/mobile/services/ServiceConstants.kt [hunk 2]
```diff
@@ -0,0 +1,5 @@
+package com.adobe.marketing.mobile.services
```
Comment (yangyansong-adbe) [inline code/android-core-library/src/phone/java/com/adobe/marketing/mobile/services/ServiceConstants.kt:1]: missing license header.

**Review by yangyansong-adbe — APPROVED** [review]
LGTM, only a couple of small comments

File: code/android-core-library/src/main/java/com/adobe/marketing/mobile/internal/eventhub/EventHub.kt (hunk 1 above)
Comment (praveek) [reply code/android-core-library/src/main/java/com/adobe/marketing/mobile/internal/eventhub/EventHub.kt:139]: I moved it to registerExtensions() API.

**Review by prudrabhat — APPROVED** [review]

**Pull request merged**